app/templates/rule_delete_modal.html

```diff
@@ -10,7 +10,13 @@ <h3 class="modal-title">Are you sure you want to delete this rule?</h3>
     <dd>{{ rule.channel }}</dd>
     <dt>Mapping:</dt>
     <dd>{{ rule.mapping }}</dd>
-  </dl>
+    <dt>Comment:</dt>
+    <dd>{{ rule.comment }}</dd>
+    <dt>Priority:</dt>
+    <dd>{{ rule.priority }}</dd>
+    <dt>Update type:</dt>
+    <dd>{{ rule.update_type }}</dd>
```

Comment:
Thanks for this! Let's just stick with comment for now...priority and update_type aren't useful information to have when deleting a rule.